proc/process.c

```diff
@@ -1377,6 +1377,10 @@ static void process_vforkThread(void *arg)
 		hal_cpuTlsSet(&current->tls, current->context);
 	}
 
+	if (proc_resourcesCopy(parent->process) < 0) {
```

Comment:
Move this before `hal_cpuDisableInterrupts()` - there's no need to keep interrupts disabled during copy, but it makes system less responsive. Access to the parent resources is synchronized by `proc_resourcesCopy` itself.

Reply:
Also, it might cause massive deadlock, as we might block on a taken mutex with interrupts disabled